app/pages/privacy-policy.spec.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import PrivacyPolicy from './privacy-policy';
+
+describe('PrivacyPolicy', () => {
+  let wrapper;
+
+  beforeEach(() => {
```

Comment:
I don't think this needs to be a `beforeEach`. `beforeEach` runs before each test in the describe block and I don't think these unit tests need the wrapper to be recreated each time.